docs/ref/settings.txt

```diff
@@ -2120,6 +2122,12 @@ affect them.
     startproject <startproject>` creates a unique ``SECRET_KEY`` for
     convenience.
 
+.. versionchanged:: 2.1
+
+   ``SECRET_KEY`` is no longer defined by default. Django will now raise an
```

Comment:
I think this is sufficient, "In older versions, SECRET_KEY defaults to an empty string." (use 4 space indent)